include/libdnf/advisory/advisory_set.hpp

```diff
@@ -127,7 +133,9 @@ class AdvisorySet {
 private:
     friend AdvisorySetIterator;
     friend class AdvisoryQuery;
+    friend class libdnf::rpm::PackageQuery;
```

Comment:
This one also doesn't seem actually used in this commit.

Reply:
The `PackageQuery` is using `get_advisory_packages_sorted_by_id`.